CodePurpose/political_leaning_api
# Augment user table

Test plan:
- run `mix deps.get`
- run 'mix ecto.reset`
- run `iex -S mix`
- in iex create a map like this: `user_params = %{
      username: "JaneyBabyyy",
      name: "Jane Smith",
      email: "jane@smith.com",
      password: "correcthorsebatterystaple"
    }`
- call User.changeset(%User{}, user_params)
- verify that the changeset is valid.
- remove required fields like username, name, email, password and verify that the changeset is invalid
